src/main/java/uk/gov/hmcts/reform/hmc/helper/hmi/HmiSubmitHearingRequestMapper.java

```diff
@@ -36,4 +46,20 @@ public HmiSubmitHearingRequest mapRequest(Long hearingId, HearingRequest hearing
             .build();
     }
 
+    /**
+     * map Delete hearing Request to HMI Submit Delete Hearing Request.
+     * @param hearingRequest delete hearing request
+     * @return hmiSubmitDeleteHearingRequest HMI Submit Delete Hearing request object
+     */
+    public HmiSubmitDeleteHearingRequest mapRequest(DeleteHearingRequest hearingRequest) {
+
+        HmiDeleteHearingRequest hmiDeleteHearingRequest = HmiDeleteHearingRequest.builder()
+                .cancellationReason(new CancellationReason(hearingRequest.getCancellationReasonCode()))
+                .build();
+
+        return HmiSubmitDeleteHearingRequest.builder()
+                .hearingRequest(hmiDeleteHearingRequest)
```

Comment:
`hearingRequest` is not part of the HMI spec for Delete, only Submit. The example request is
```
{
    "_cancel": {
        "cancellationReasonCode": "CNCL"
    }
}  
```

whereas this generates
```
{
    "hearingRequest": {
        "_cancel": {
            "cancellationReasonCode": "CNCL"
        }
    } 
}  
```

It looks like we can probably just return the previously built `HmiDeleteHearingRequest` and delete this extra `HmiSubmitDeleteHearingRequest` object completely and it'll give the desired result.